src/app/containers/FrontPageMain/index.jsx

```diff
@@ -32,7 +35,13 @@ const FrontPageMain = ({ frontPageData }) => {
     <>
       <ATIAnalytics data={frontPageData} />
       <ChartbeatAnalytics data={frontPageData} />
-      <MetadataContainer metadata={metadata} promo={promo} />
+      <MetadataContainer
+        title={frontPageTitle}
+        lang={metadata.language}
```

Comment:
Can you use pathOr around these please? if metadata disapears the app with crash